requirements/extra.txt

```diff
@@ -9,5 +9,5 @@ onnxruntime>=1.3.0
 hydra-core>=1.0
 # todo: when switch to standard package stream, drop `fairscale` from hard mocked docs libs
 https://github.com/PyTorchLightning/fairscale/archive/pl_1.2.0.zip
-jsonargparse[signatures]>=3.3.1
+jsonargparse[signatures]>=3.9.0
```

Comment:
this is for `fail_untyped` and `required`, right?

Reply:
Yes. Though more specifically to have a simple way to implement the backbone in backbone_image_classifier.py since without this it would have become unnecessarily complex.